src/SwcCompiler.ts

```diff
@@ -207,7 +207,7 @@ export class SwcCompiler implements Compiler {
 
   /** The list of globby patterns to use when searching for files to build */
   private fileGlobPatterns(config: ProjectConfig) {
-    const extensions = config.esbuild?.resolveExtensions || DefaultExtensions;
+    const extensions = config.extensions || DefaultExtensions;
```

Comment:
This changes things a little. We already have a `config.extension` key, but the `config.esbuild` object also allowed defining extensions to scan. Is this change ok?